src/Microsoft.HttpRepl/Microsoft.HttpRepl.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk;Microsoft.DotNet.GlobalTools.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <PackAsTool>true</PackAsTool>
+    <AssemblyName>dotnet-httprepl</AssemblyName>
+    <LangVersion>latest</LangVersion>
+    <Description>Command line tool to for making HTTP calls and viewing their results.</Description>
+    <PackageTags>dotnet;http;httprepl</PackageTags>
+
+    <!-- This is a requirement for Microsoft tool packages only. -->
+    <GenerateToolShims>true</GenerateToolShims>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNet.WebApi.Client" Version="$(MicrosoftWebApiClientPackageVersion)" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Microsoft.Repl\Microsoft.Repl.csproj" />
+  </ItemGroup>
+  
+  <ItemGroup>
+    <SignedPackageFile Include="tools/$(TargetFramework)/any/Microsoft.Repl.dll" Certificate="$(AssemblySigningCertName)" />
+    <SignedPackageFile Include="tools/$(TargetFramework)/any/Newtonsoft.Json.dll" Certificate="$(AssemblySigningCertName)" />
```

Comment:
Can you change third party binaries to use `Certificate="Microsoft3rdPartyAppComponentDual"`. If it uses `$(AssemblySigningCertName)`, it would cause us to sign JSON.NET with the Microsoft cert, which is wrong. I see dotnet-user-secrets.csproj also has this wrong. We're not actually using this metadata yet, but will be in the next few weeks.